docs/how-to/contribute-code.rst

```diff
@@ -52,8 +52,17 @@ Install Node.JS
 ^^^^^^^^^^^^^^^
 
 Lastly, you'll need to install `Node.js`_. You need to have a recent version
-of Node; we test using v6.9.1. Once you've installed node, you can use it to
-install Batavia's JavaScript dependencies, and compile the Batavia library:
+of Node; we test using v6.9.1. One civilized way to get an isolated Node.js
```

Comment:
"civilized" is a little pejorative; also, if we're going to make the recommendation, the docs should commit to it - "We suggest using `nodeenv`", rather than "one way is to...".

Should we also add the `--node=6.9.1` flag to ensure the right version is available?

Reply:
I have a hard time committing to this suggestion, because my experience with node.js is rather limited. I agree to a less opinionated phrasing, we can drop "civilized".

With respect to the node version -- adding a specific one sounds like a good idea, but I've been working with the default (latest, currently v10.2.1) and there don't seem to be issues. Should we perhaps update?